modules/lazyload/module.php

```diff
@@ -0,0 +1,119 @@
+<?php
+namespace Elementor\Modules\LazyLoad;
+
+use Elementor\Core\Base\Module as BaseModule;
+use Elementor\Core\Experiments\Manager as Experiments_Manager;
+use Elementor\Element_Base;
+use Elementor\Plugin;
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Module extends BaseModule {
+
+	const EXPERIMENT_PREFIX = 'elementor_experiment-';
+	const EXPERIMENT_NAME = 'e_lazy_load_background_images';
+
+	public function get_name() {
+		return 'lazyload';
+	}
+
+	public static function get_experimental_data() {
+		return [
+			'name' => static::EXPERIMENT_NAME,
+			'title' => esc_html__( 'Lazy Load Background Images', 'elementor' ),
+			'tag' => esc_html__( 'Performance', 'elementor' ),
+			'description' => esc_html__( 'Lazy loading images that are not in the viewport improves initial page load performance and user experience. By activating this experiment all background images except the first one on your page will be lazy loaded to improve your LCP score', 'elementor' ),
+			'release_status' => Experiments_Manager::RELEASE_STATUS_BETA,
+			'default' => Experiments_Manager::STATE_INACTIVE,
+		];
+	}
+
+	public function enqueue_scripts() {
+		wp_enqueue_script(
+			'elementor-lazyload',
+			$this->get_js_assets_url( 'lazyload' ),
+			[ 'elementor-frontend' ],
+			ELEMENTOR_VERSION,
+			true
+		);
+	}
+
+	public function enqueue_styles() {
+		wp_enqueue_style(
+			'elementor-lazyload',
+			$this->get_css_assets_url( 'modules/lazyload/frontend' ),
+			[],
+			ELEMENTOR_VERSION
+		);
+	}
+
+	private function update_element_attributes( Element_Base $element ) {
+		$settings = $element->get_settings_for_display();
+		$controls = $element->get_controls();
+		$keys = false;
+
+		$controls_with_background_image = array_filter( $controls, function( $control ) {
+			if ( Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'active' ] ) ) {
+				return true;
+			}
+		} );
+
+		// For now, we only support the first control within an element that has lazyload enabled,
+		// we will support more controls in the same element that used lazyload in the future.
```

Comment:
```suggestion
		// For now, it only support the first control within an element that has lazyload enabled,
		// it will support more controls in the same element that used lazyload in the future.
```